save-orchestrator-common/src/test/kotlin/com/saveourtool/save/orchestrator/docker/DockerContainerManagerTest.kt

```diff
@@ -53,7 +53,7 @@ class DockerContainerManagerTest {
         val testFile = createTempFile().toFile()
         testFile.writeText("wow such testing")
         dockerAgentRunner.createAndStart(
-            executionId = 42,
+            executionId = 43,
```

Comment:
how this change fix `flacky`?

Reply:
42 is used in AgentsControllerTest, it's not always managed to shutdown container so the next test can't use it and failed.

Reply:
suggestion to generate such ids instead of using it hardcoded

Reply:
but as I remember, we used `42` more, than in one (current) place, isn't it? shouldn't we change other places too?

Reply:
i found only 2 files